assets/src/scripts/components/EmptySearch.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+export default function EmptyState() {
+  return (
+    <section style={{ maxWidth: "80ch" }}>
+      <p className="lead">Your search returned no concepts. You may want to:</p>
+      <ol>
+        <li>Double check the spelling</li>
+        <li>
+          Make sure that if you're searching for a code (rather than a search
+          term) that you have selected the correct option ("Search by code"
+          above the search box on the left)
+        </li>
+        <li>Try searching for a similar term</li>
+      </ol>
+      <p>
+        If this search was an error, you can click "Remove" from the "Previous
+        searches" box on the left to get rid of it. However, if you want other
+        people to know that you tried this search, but it returned no results,
+        then you can leave it as a blank search.
+      </p>
```

Comment:
```suggestion
        <li>Check your spelling</li>
        <li>Try searching for a similar term</li>
        <li>
            Ensure you have selected the correct option above the search box
            ("Search by code" if searching by code rather than term)
        </li>
      </ol>
      <p>
        If this search was an error, click "Remove" in the "Previous searches"
        box on the left to delete it. If you'd like to record that you
        performed this search but it returned no results, you can leave it as
        is.
      </p>
```
- avoiding "double check" and "get rid of" as slightly tricky for non-native speakers
- putting the two bullets about term-based search first and together
- slight rephrasings for clarity / brevity

Reply:
Very nice - thanks!